tlatools/org.lamport.tlatools/src/pcal/ParseAlgorithm.java

```diff
@@ -842,7 +843,8 @@ public static AST.VarDecl GetVarDecl() throws ParseAlgorithmException
        pv.col  = lastTokCol ;
        pv.line = lastTokLine ;
        if (   PeekAtAlgToken(1).equals("=")
-           || PeekAtAlgToken(1).equals("\\in"))
+           || PeekAtAlgToken(1).equals("\\in")
+           || PeekAtAlgToken(1).equals("∈"))
```

Comment:
Here (and in a few other places) it looks like we're duplicating information from `ConfigConstants.OperatorSynonyms`. Is it at all practical to deduplicate that information? E.g. this check would be something like `Operators.getOperator(PeekAtAlgToken(1)).equals(Operators.getOperator("\\in"))`, which covers both the `\in` and `∈` cases.

Reply:
Yeah I have that desire as well, a sort of central repository of all information about operators. I think it's viable but I would want test coverage to be better (currently considering how to improve test coverage of the pluscal code generally).